src/sentinel/azext_sentinel/default_detection_template.yaml

```diff
@@ -42,6 +42,33 @@ suppression_enabled: True
 # example - playbook_name: 'PostAlertActions-SentinelTrigger'
 playbook_name: <PLACEHOLDER>
 
+# Incident configuration settings when this alert rule is triggered
```

Comment:
s/alert rule/detection  (users shouldn't be exposed to the implementation name "alert rule")